src/applications/simple-forms/21-0966/containers/ConfirmationPage.jsx

```diff
@@ -7,31 +7,139 @@ import scrollToTop from 'platform/utilities/ui/scrollToTop';
 import { focusElement } from 'platform/utilities/ui';
 import FormFooter from 'platform/forms/components/FormFooter';
 
+import { ConfirmationView } from 'platform/forms-system/src/js/components/ConfirmationView';
+import { DevOnlyTestVariations } from '../components/DevOnlyTestVariations';
 import GetFormHelp from '../../shared/components/GetFormHelp';
 import {
   hasActiveCompensationITF,
   hasActivePensionITF,
   confirmationPageFormBypassed,
   confirmationPageAlertStatus,
   confirmationPageAlertHeadline,
+  confirmationPageAlertHeadlineV2,
   confirmationPageAlertParagraph,
   confirmationPageNextStepsParagraph,
+  confirmationPageAlertParagraphV2,
 } from '../config/helpers';
 import {
   veteranBenefits,
   survivingDependentBenefits,
+  submissionApis,
 } from '../definitions/constants';
+import { useNewConfirmationPage } from '../config/features';
+
+const NextSteps = ({ formData }) => (
+  <div>
+    <h2>What are my next steps?</h2>
+    {confirmationPageNextStepsParagraph(formData) ? (
+      <>
+        <p>You should complete and file your claims as soon as possible.</p>
+        <p>{confirmationPageNextStepsParagraph(formData)}</p>
+      </>
+    ) : (
+      <p>
+        You should complete and file your claims as soon as possible. If you
+        complete and file your claim before the intent to file expires and we
+        approve your claim, you may be able to get retroactive payments.
+        Retroactive payments are payments for the time between when we processed
+        your intent to file and when we approved your claim.
+      </p>
+    )}
+  </div>
+);
+
+const ActionLinksToCompleteClaims = ({ formData }) => (
```

Comment:
Extracted this from existing code below. Only new change is to use `va-link-action`